File: Training/URI/categoria-Iniciante/java/1175.java
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		int[] vetor = new int[20];
		
		for(int i = 0; i < 20; i++) {
			vetor[i] = scanner.nextInt();
		}
		
		int j = 19;
		int aux;
		for(int i = 0; i < 10; i ++) {
			aux = vetor[i];
			vetor[i] = vetor[j];
			vetor[j] = aux;
			j--;
		}
		
		for(int i = 0; i < 20; i ++) {
			System.out.println("N[" + i +"] = " + vetor[i]);
		}
		
		
		
	}
}

File: Training/URI/categoria-Iniciante/C/1175.c
#include<stdio.h>

int main() {

    int vetor[20];
    
    int i;
    
    for(i = 0; i < 20; i++) {
        scanf("%d",&vetor[i]);
    }
    
    int j = 19;
		int aux;
		for(i = 0; i < 10; i ++) {
			aux = vetor[i];
			vetor[i] = vetor[j];
			vetor[j] = aux;
			j--;
		}
		
		for(i = 0; i < 20; i++){
            printf("N[%d] = %d\n",i, vetor[i]);
        }
		
}
